package.json

```diff
@@ -134,7 +135,7 @@
     "esbuild": "^0.20.2",
     "eslint": "^8.14.0",
     "eslint-config-airbnb": "^19.0.4",
-    "eslint-plugin-import": "^2.28.1",
+    "eslint-plugin-import": "^2.29.1",
```

Comment:
I was trying to fix this by bumping it since it does not cause issues lets just update it now